src/redux/reducers/user/index.js

```diff
@@ -1,18 +1,15 @@
 import {
-  GET_USER_LOCAL_STORAGE,
   GET_USER_INFO
 } from './../../actions/user/types'
+
 import * as user from './reducers'
 
 const INITIAL_STATE = {
-  user: {},
-  info: 'Init Mapping'
+  user: {}
 }
 
 export default (state = INITIAL_STATE, action) => {
   switch (action.type) {
-    case GET_USER_LOCAL_STORAGE:
-      return user.setUser(state, action)
     case GET_USER_INFO:
```

Comment:
faz sentido usar `GET` para um reducer action? de fato isso faz `GET` de algo? ou todas as acoes fazem get?

Reply:
The old problem 'finding name'